internal/buildmanifest/manifest.go

```diff
@@ -0,0 +1,66 @@
+package buildmanifest
+
+import "gopkg.in/yaml.v2"
+
+const (
+	// Filename of a build manifest.
+	Filename = "build.yml"
+	// CurrentVersion os the manifest file format. See Manifest.SDK.
```

Comment:
```suggestion
	// CurrentVersion is the manifest file format. See Manifest.SDK.
```

I would also suggest naming this `FileFormat` rather than "Version" since we have a lot of different version markers in play here (and `buildmanifest.FileFormat` reads pretty well at the usage site).